CHANGELOG.md

```diff
@@ -226,13 +226,13 @@ In addition to a bunch of bug fixes, we've made two improvements to the gateway.
 
 You can play with both of these features by visiting:
 
-> http://bafybeia6po64b6tfqq73lckadrhpihg2oubaxgqaoushquhcek46y3zumm.ipfs.io/
+> http://bafybeia6po64b6tfqq73lckadrhpihg2oubaxgqaoushquhcek46y3zumm.ipfs.localhost:8080
```

Comment:
I'd use dweb.link (see #7240). That way, this works for users without IPFS installed.

Reply:
@Stebalien  Listings are still broken there, that is why I explicitly did not use it as an example.

Reply:
https://bafybeia6po64b6tfqq73lckadrhpihg2oubaxgqaoushquhcek46y3zumm.ipfs.dweb.link/ipfs/bafybeia6po64b6tfqq73lckadrhpihg2oubaxgqaoushquhcek46y3zumm/quick-start → error

Reply:
Fix requires https://github.com/protocol/bifrost-infra/pull/573 to land